### ISSUE TITLE
Multiline comments incorrectly highlighted

### ISSUE THREAD

**Issue by pchaigno** [michaeledgar/protobuf-tmbundle#2]
Multiline comments (with `/*` and `*/`) are not highlighted: [example in Lightshow](https://github-lightshow.herokuapp.com/?utf8=%E2%9C%93&scope=from-url&grammar_url=https%3A%2F%2Fraw.githubusercontent.com%2Fmichaeledgar%2Fprotobuf-tmbundle%2Fmaster%2FProtocol%2520Buffers.tmbundle%2FSyntaxes%2FProtocol%2520Buffer.tmLanguage&grammar_text=&code_source=from-text&code_url=&code=%2F**%0D%0A+*+Licensed+to+the+Apache+Software+Foundation+%28ASF%29+under+one%0D%0A+*+or+more+contributor+license+agreements.++See+the+NOTICE+file%0D%0A+*+distributed+with+this+work+for+additional+information%0D%0A+*+regarding+copyright+ownership.++The+ASF+licenses+this+file%0D%0A+*+to+you+under+the+Apache+License%2C+Version+2.0+%28the%0D%0A+*+%22License%22%29%3B+you+may+not+use+this+file+except+in+compliance%0D%0A+*+with+the+License.++You+may+obtain+a+copy+of+the+License+at%0D%0A+*%0D%0A+*+++++http%3A%2F%2Fwww.apache.org%2Flicenses%2FLICENSE-2.0%0D%0A+*%0D%0A+*+Unless+required+by+applicable+law+or+agreed+to+in+writing%2C+software%0D%0A+*+distributed+under+the+License+is+distributed+on+an+%22AS+IS%22+BASIS%2C%0D%0A+*+WITHOUT+WARRANTIES+OR+CONDITIONS+OF+ANY+KIND%2C+either+express+or+implied.%0D%0A+*+See+the+License+for+the+specific+language+governing+permissions+and%0D%0A+*+limitations+under+the+License.%0D%0A+*%2F%0D%0A%0D%0Apackage+mesos%3B).

**Comment by pchaigno** [comment]
Other bug as a consequence:
A `'` in what should be recognized as a comment break highlighting of following lines:
[example in Lightshow](https://github-lightshow.herokuapp.com/?utf8=%E2%9C%93&scope=from-url&grammar_url=https%3A%2F%2Fraw.githubusercontent.com%2Fmichaeledgar%2Fprotobuf-tmbundle%2Fmaster%2FProtocol%2520Buffers.tmbundle%2FSyntaxes%2FProtocol%2520Buffer.tmLanguage&grammar_text=&code_source=from-text&code_url=&code=message+CommandInfo+{%0D%0A%2F%2F+Command+health+check.%0D%0A++optional+CommandInfo+command+%3D+7%3B%0D%0A}%0D%0A%0D%0A%0D%0A%2F**%0D%0A+*+%28currently+[compressed]+tar+and+zip%29+it+is+extracted+into+the+executor%27s%0D%0A+*%2F%0D%0Amessage+CommandInfo+{%0D%0A++message+URI+{%0D%0A++++required+string+value+%3D+1%3B%0D%0A++++optional+bool+executable+%3D+2%3B%0D%0A++}%0D%0A}).

**Comment by pchaigno** [comment]
@michaeledgar Did you get a chance to look into this?

**Comment by parkr** [comment]
@pchaigno Lightshow is down for maintenance. If you can paste an example here, I can try to get a patch in my fork.

**Comment by parkr** [comment]
Multiline comments: https://github.com/parkr/protobuf-tmbundle/commit/713e0d8ccbfa0eab0fd0d8d3c369df6b008b5cf1

**Comment by parkr** [comment]
@pchaigno I'm not sure what you mean by this:

> A ' in what should be recognized as a comment break highlighting of following lines:

**Comment by pchaigno** [comment]
It's great that you're working on this! Thanks!

To be honest, I don't remember these issues and it's likely Lightshow is the only place I've stored the code snippets to reproduce the bugs. Any chance you could reuse the URIs directly? Otherwise, I can try to extract it from the GET parameter :-)

**Comment by parkr** [comment]
Extracted the second to:

```protobuf
message CommandInfo {
// Command health check.
  optional CommandInfo command = 7;
}


/**
 * (currently [compressed] tar and zip) it is extracted into the executor's
 */
message CommandInfo {
  message URI {
    required string value = 1;
    optional bool executable = 2;
  }
}
```

**Comment by parkr** [comment]
Locally, this works:

<img width="786" alt="screen shot 2018-08-08 at 3 59 53 pm" src="https://user-images.githubusercontent.com/237985/43861200-1c3dfcf0-9b24-11e8-917a-38530e42f8bf.png">

**Comment by pchaigno** [comment]
Hm, yeah I guess handling the first issues automatically takes care of the second :smiley: 
(It was probably, not seing `/*` as the start of a multiline comment, the parser saw `'` as the start of a new string and highlighted all subsequent code as such. Only guessing here.) 

Looks all fixed to me!

**Comment by parkr** [comment]
@pchaigno Great! I reverted the removal of `required` and `optional` too so that works.

Is there a good way to submit #3 upstream to github/linguist?

**Comment by pchaigno** [comment]
> Is there a good way to submit #3 upstream to github/linguist?

No need, grammars are updated automatically with each new release.

**Comment by parkr** [comment]
@pchaigno ok! If @michaeledgar doesn't respond, I'd be happy to sacrifice myself as the maintainer for a little while and you can update linguist to pull from my fork.

**Comment by pchaigno** [comment]
Sure!

**Comment by michaeledgar** [comment]
Merged in #3 which should fix this IIUC.
